samples/python-flask/13.core-bot/main.py

```diff
@@ -0,0 +1,59 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""
+This sample shows how to create a simple EchoBot with state.
```

Comment:
Not EchoBot, we should change this in the other sample as well.

Reply:
I'll fix this on the next checkin - coming shortly..